README.md

```diff
@@ -96,6 +96,17 @@ fizz.Header(name, desc string, model interface{})
 // Override the binding model of the operation.
 fizz.InputModel(model interface{})
 
+// Add a security requirement to the list of security overrides for this operation.
+// Note that this removes all other SecurityRequirements
+// that might have been specified in the root `security` property.
+fizz.Security(security *openapi.SecurityRequirement)
+
+// Make other security requirements optional.
```

Comment:
Again, I think it is better to be explicit. We should indicate what the function does, not just its effect:

> Add an empty security requirement to this operation to make other security requirements optional.

Reply:
I was not sure how to break this one - let me know if you want me to reformat it:)